pages/benchmark/index.html

```diff
@@ -112,7 +111,7 @@
   <h2>Partition Benchmarks</h2>
   <div id="partitions" class="chart"></div>
   <div id="partitions_labels" class="cr-legend"></div>
-  <p id="partitions_descripton" class="description" data-log-scale="true">
```

Comment:
@chaudum I removed it from the partitions benchmarks because it doesn't make it more readable